### DISCUSSION TITLE
Setoption111 on ESP32

### DISCUSSION THREAD

**Discussion by storchalberto** [arendst/Tasmota#18583] [Q&A]
Using tasmota32.bin the setoption111 1 do not function on esp32. The same command work correctly on an esp8266 board (tasmota.bin). I have done the test using the command buzzer 5,50,50. Using setoption 111 0  and buzzer 5,50,50 on both the device i can see the pin correctly go on and off for the timing programmed. if i do the command setoption111 1, and then buzzer 5,50,50, on the esp8266 i see for the 5 second  a square wave (for 5 times), on the esp32 i see the signal on for 5 times. The pin on ESP32 works correctly, i have tested it (GPIO21) configuring this pin as pwm and i can  see a square wave, so it seems to be a sw problem on the esp32.

**Comment by Jason2866** [comment]
Probably this `analogAttach(Pin(GPIO_BUZZER), 7);` does not work, because of this bug https://github.com/espressif/arduino-esp32/issues/8106 in Arduino core.
Will be fixed, when Tasmota Core 2.0.9 will be released.
